### PR TITLE
tests: fix tests for go-ftw

### PR DESCRIPTION
**Pull request by fzipi**
Signed-off-by: Felipe Zipitria <felipe.zipitria@owasp.org>

- moved status to int list
- changed data string lists to multiline

### CODE REVIEW

File: tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920181.yaml [hunk 1]
```diff
@@ -1,35 +1,31 @@
...
+              Host: "localhost"
+              Accept: "*/*"
+              Content-Length: 7
+              Content-Type: "application/x-www-form-urlencoded"
+              Transfer-Encoding: "chunked"
+              User-Agent: "ModSecurity CRS 3 Tests"
+            data: |
+              7
+              foo=bar
+              0
```
Comment (theseion) [inline tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920181.yaml:26]: Careful! I think those two additional line breaks might have meaning!

**Comment by dune73** [comment]
I do not understand nearly enough to review YAML format tests. @theseion are you on this?

**Comment by theseion** [comment]
Yes.

File: tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920181.yaml (hunk 1 above)
Comment (fzipi) [reply tests/regression/tests/REQUEST-920-PROTOCOL-ENFORCEMENT/920181.yaml:26]: Oops. They might, but tests are still passing :D Re-adding.

**Review by theseion — APPROVED** [review]

**Pull request merged**